src/Components/TopNavi.js

```diff
@@ -79,13 +79,13 @@ const TopNavi = props => {
       <Right>
         <A href="#">whitepaper</A>
         <MobileNone>
-          <StyledLink to="/charts">charts</StyledLink>
+          {/* <StyledLink to="/charts">charts</StyledLink> */}
         </MobileNone>
         <MobileNone>
           <StyledLink to="/Investment">investment</StyledLink>
         </MobileNone>
         <MobileNone>
-          <A href="#">discord</A>
+          <A href="https://discord.gg/Mnda9wr">discord</A>
```

Comment:
Link to discord route instead of directly

Reply:
+1